README.md

```diff
@@ -101,6 +101,10 @@ The `scripts` folder in this repository contains scripts that interact with the
 
   Checks all free-text fields in G-Cloud services for "bad words" and generates a CSV report of any bad words found.
 
+* `send_dos_opportunities_email.py`
```

Comment:
If you don't think it is a quick fix, then yes we should make a record of it somewhere. Tech debt is probably the most appropriate place to put it. 👍